### DISCUSSION TITLE
Checkbox readonly problem

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#10029] [Help]
How to make checkbox become readonly or, at least it looks like readonly

I tried using disabled(), but it make the color not same (become blur)
```
    Checkbox::make('useBox')->label('Use Box')
        ->inline(false)
        ->default(false),

    Checkbox::make('useBox')->label('use Box')
        ->inline(false)
        ->disabled()
        ->default(false),
```
when the value is true.
![image](https://github.com/filamentphp/filament/assets/6214964/a3303275-a1a9-4f3d-a531-0ed8c26d175a)

when the value is false.
![image](https://github.com/filamentphp/filament/assets/6214964/4bcfc75f-c110-4284-8d21-83bf3c1b25b1)

Using readonly() make it become BadMethodCallException.
```
      Checkbox::make('useBox')->label('use Box')
          ->inline(false)
          // ->disabled()
          -> readonly()
          ->default(false),
```
![image](https://github.com/filamentphp/filament/assets/6214964/ce2dfa2b-6241-42ce-8ee1-a60deeba1c2b)

I need advice. Thanks a lot.

**Comment by leandrocfe** [comment]
You can use `disabled()` and [customize a theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) to change what you want.

**Reply by jigujigu** [reply]
Thanks,

But, I don't know how to customize a theme.
Can you please explain or make a sample for me ?

Thanks

**Reply by leandrocfe** [reply]
https://filamentphp.com/docs/3.x/panels/themes/#creating-a-custom-theme

?
